lib/cn/csched_sp3.c

```diff
@@ -2436,7 +2441,8 @@ sp3_create(
     mutex_init(&sp->new_tlist_lock);
     mutex_init(&sp->work_list_lock);
 
-    cv_init(&sp->cv, "csched");
+    mutex_init(&sp->mon_lock);
+    cv_init(&sp->mon_cv, "csched");
```

Comment:
It's rather suprising that it worked given we weren't initializing the monitor mutex!!!